File: conference/management-bc/web/src/main/java/com/microsoft/conference/management/commands/UpdateConference.cs
using System;
using ENode.Commanding;

namespace ConferenceManagement.Commands
{
    public class UpdateConference : Command<Guid>
    {
        public string Name { get; set; }
        public string Description { get; set; }
        public string Location { get; set; }
        public string Tagline { get; set; }
        public string TwitterSearch { get; set; }
        public DateTime StartDate { get; set; }
        public DateTime EndDate { get; set; }
    }
}

File: samples/conference/common/src/main/java/com/microsoft/conference/common/management/commands/UpdateConference.java
package com.microsoft.conference.common.management.commands;

import org.enodeframework.commanding.Command;

import java.util.Date;

public class UpdateConference extends Command<String> {
    public String name;
    public String description;
    public String location;
    public String tagline;
    public String twitterSearch;
    public Date startDate;
    public Date endDate;
    public Boolean isPublished;
}
